### ISSUE TITLE
Dev build blocked by Windows Defender

### ISSUE THREAD

**Issue by WrayOfSunshine** [DevilXD/TwitchDropsMiner#634]
Windows Defender thinks the current dev build zip has a trojan, specifically "win32/suschil!rfn"

**Comment by DevilXD** [comment]
This is a duplicate of #620, #616, #517, #508, #503, #420, #384, #335, #333, #315, #284, #270, #249, #217, #209, #137 and #13. Please read the search the existing issues before submitting new ones.
